XenAdmin/Controls/BondDetails.cs

```diff
@@ -279,6 +280,22 @@ internal DialogResult ShowCreationWarning()
                 }
                 return dialogResult;
             }
+
+            if (will_disturb_clustering)

```

Comment:
Also, is the order of warnings meaningful (i.e. why has it been inerted between primary and secondary interface check)?